lib/load-paths-handler.coffee

```diff
@@ -74,13 +75,12 @@ class PathLoader
         done
       )
 
-  isInternalSymlink: (pathToLoad) ->
-    realPath = null
-    try
-      realPath = fs.realpathSync(pathToLoad)
-    catch err
-      ; # ignore a broken symlink
-    realPath?.search(@rootPath) is 0
+  isInternalSymlink: (pathToLoad, done) ->
+    fs.realpath pathToLoad, @realPathCache, (err, realPath) =>
+      if err
+        done(false)
+      else
+        done(realPath?.search(@rootPath) is 0)
```

Comment:
I don't think `realPath` can be `null` at this point so the `?` is probably no longer needed.